metrics.py

```
@@ -14,12 +14,40 @@
 
 import datadog
 import os
+import boto3
 from common import *
+from botocore.exceptions import ClientError
 
+def get_datadog_api_key():
+    secret_name = os.environ.get("DATADOG_API_KEY_SECRET_NAME")
+    region_name = os.environ.get("AWS_REGION", "")
+
+    if not region_name:
+        print("Can't fetch the current AWS region")
+        return
+
+    session = boto3.session.Session()
+    client = session.client(
+        service_name='secretsmanager',
+        region_name=region_name
+    )
+
+    try:
+        get_secret_value_response = client.get_secret_value(
+            SecretId=secret_name
+        )
+    except ClientError as e:
```

I'd just let it fail without catching.